frontend/src/components/CreateYourAccount/index.js

```diff
@@ -0,0 +1,203 @@
+import React, { useState } from "react";
+import {
+  Button,
+  Checkbox,
+  Hero,
+  Input,
+  RadioList,
+  Select
+} from "@code4ro/taskforce-fe-components";
+import BasePage from "../BasePage";
+import SidebarLayout from "../SidebarLayout";
+import "./styles.scss";
+import api from "../../api";
+
+const CreateYourAccount = () => {
+  const [form, setForm] = useState({
+    fullName: { value: "", isValid: false, isDirty: false },
+    age: { value: undefined, isValid: false, isDirty: false },
+    gender: { value: 0, isValid: false, isDirty: false },
+    preexistingMedicalCondition: { value: "", isValid: false, isDirty: false },
+    quarantineStatus: { value: undefined, isValid: false, isDirty: false },
+    agreeToTerms: { value: false, isValid: false, isDirty: false },
+    isFormValid: false
+  });
+
+  const validation = (name, value) => {
+    switch (name) {
+      case "fullName":
+        return !!value && value.length < 100;
+      case "age":
+        return !!+value && +value > 0 && +value < 100;
+      case "gender":
+        return +value !== 0;
+      case "preexistingMedicalCondition":
+        return !value || value.length < 500;
+      case "quarantineStatus":
+        return !!value;
+      case "agreeToTerms":
+        return value;
+      default:
+        return false;
+    }
+  };
+
+  const submitForm = () => {
+    if (!form.isFormValid) {
+      return;
+    }
+
+    api.post(`http://localhost:5008/api/user`, {
```

Comment:
Can we use the API_URL here? This will only work in case of a local development environment.